pkg/util/ip/addr.go

```diff
@@ -43,3 +43,21 @@ func lastIP(subnet net.IPNet) net.IP {
 	}
 	return end
 }
+
+// lastUsableIP returns second to last IP of a subnet
+func LastUsableIP(subnet net.IPNet) net.IP {
+	// FIXME(dulek): This should have proper IPv6 support, but for now whatever, Kuryr doesn't support it yet.
+	ip := lastIP(subnet)
+	// And this second to last (last usable)
+	ip[len(subnet.IP)-1] &= 0xFE
```

Comment:
The last IP is guaranteed to be an odd number, so `& 0xFE` just means `- 1`

Also `len(subnet.IP)` should really be `len(ip)`. Also, the comment is ungrammatical. ("And this second to last")